detection-rules/link_russian_traversed_credential_phishing.yml

```diff
@@ -0,0 +1,53 @@
+name: "Link: Credential Phishing via messages with Russian TLD traversal"
```

Comment:
```suggestion
name: "Link: Credential Phishing via Russian sending infrastructure"
```

Reply:
I thought about that too, sending infrastructure seems like we're saying origination when this is really just any route traversal.